homeassistant/components/lifx/coordinator.py

```diff
@@ -219,6 +222,28 @@ async def async_set_color_zones(
             )
         )
 
+    async def async_update_multizone_effect(self) -> None:
+        """Update the device firmware effect running state."""
+        await async_execute_lifx(self.device.get_multizone_effect)
+        self.move_effect_active = self.device.effect.get("effect", None) == "MOVE"
```

Comment:
Can we make this `self.active_effect` instead so we don't end up with a lot of these if there are more ?

Reply:
I refactored this to use an `Enum` so that it supports the Tile effects ('MORPH' and 'FLAME') as well.